scipy/stats/_continuous_distns.py

```diff
@@ -8703,6 +8703,15 @@ def _logsf(self, x, a, b):
             logsf[i] = np.log1p(-np.exp(self._logcdf(x[i], a[i], b[i])))
         return logsf
 
+    def _entropy(self, a, b):
+        A = sc.ndtr(a)
+        B = sc.ndtr(b)
+        Z = B - A
+        C = np.log(np.sqrt(2 * np.pi * np.e) * Z)
```

Comment:
> The flip side to that is that taking the logarithm apart puts us at risk of underflow

I don't see that.

> It's a very small difference, but calling np.log twice instead of once is slower.

Note that even if it saves an evaluation of `log`, the current implementation adds a `sqrt`. On my machine, these tend to take the same amount of time. If we're going for speed, `Z**.5` seems faster on my machine.

> It's not a huge deal

Agreed if it's between these options, but if `_log_gauss_mass` were used, adding `logZ` would be the better choice.

Reply:
>I don't see that.

I didn't mean as a realistic risk, just as a hypothetical risk, in the same way that we might hypothetically have an overflow if Z ends up being extremely large.

Reply:
I think the difference is that `np.log(np.sqrt(2 * np.pi * np.e)) + np.log(Z)` cannot underflow for real `Z`, even hypothetically. AFAICT, the closest it can possibly come is one of:

```python3
import numpy as np
np.log(np.nextafter(1, -np.inf))  # -1.1102230246251565e-16
np.log(np.nextafter(1, np.inf))  # 2.2204460492503128e-16
```

It is possible for the imaginary component of a complex `Z` to underflow (e.g. `np.log(1e100 + 1e-3001j)`), but I think we're only interested in real `Z`.